recipes/colorspacious/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "colorspacious" %}
+{% set version = "1.1.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "5e9072e8cdca889dac445c35c9362a22ccf758e97b00b79ff0d5a7ba3e11b618" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - numpy
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - colorspacious
+
+about:
+  home: https://github.com/njsmith/colorspacious
+  license: MIT
+  license_family: MIT
+  license_file: ''
+  summary: A powerful, accurate, and easy-to-use Python library for doing colorspace conversions
+  description: "colorspacious\n=============\n\n.. image:: https://travis-ci.org/njsmith/colorspacious.svg?branch=master\n   :target: https://travis-ci.org/njsmith/colorspacious\n   :alt: Automated test\
```

Comment:
@kthyng we usually remove some of the boilerplate that conda-skeleton creates to get a cleaner recipe. Let me push a commit here to show you.